src/store/enhancedImageBuilderApi.ts

```diff
@@ -62,7 +87,7 @@ const enhancedApi = imageBuilderApi.enhanceEndpoints({
           .then(() => {
             // Typescript is unaware of tag types being defined concurrently in enhanceEndpoints()
             // @ts-expect-error
-            dispatch(imageBuilderApi.util.invalidateTags(['Compose']));
+            dispatch(imageBuilderApi.util.invalidateTags(['Blueprints']));
```

Comment:
not sure, if we want this, but if we do, should we invalidate both of the tags?

Reply:
great rebase issues catch! :D